FWCore/Framework/interface/DependentRecordImplementation.h

```diff
@@ -76,6 +76,44 @@ class DependentRecordImplementation : public EventSetupRecordImplementation<Reco
         return eventSetupT.tryToGet<DepRecordT>();
       }
 
+      using EventSetupRecordImplementation<RecordT>::getHandle;
+
+      template<typename ProductT, typename DepRecordT>
+      ESHandle<ProductT> getHandle(ESGetToken<ProductT, DepRecordT> const& iToken) const {
+        //Make sure that DepRecordT is a type in ListT
+        using EndItrT = typename boost::mpl::end< ListT >::type;
+        using FoundItrT = typename boost::mpl::find< ListT, DepRecordT >::type;
+        static_assert(! std::is_same<FoundItrT, EndItrT>::value, "Trying to get a a product with an ESGetToken specifying a Record from another Record where the second Record is not dependent on the first Record.");
```

Comment:
fix duplicate "a" in this message and the two below ("Trying to get a a product")

Reply:
Good catch, thanks, fixed.